packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -302,6 +259,7 @@ sub-item-cancel-confirm =
 account-activated = Your account is activated, <userEl/>
 
 ## subscription route index
+## $date (Date) - Last day of product access
```

Comment:
Can you also document `$name` in the group comment or next to `sub-route-idx-cancel-msg`?